addons/binding/org.openhab.binding.sinope/OSGI-INF/.gitignore

```diff
@@ -0,0 +1 @@
+/org.openhab.binding.sinope.internal.SinopeHandlerFactory.xml
```

Comment:
It should be OK to ignore any `.xml` file here, i.e. `*.xml` because other annotation processors might generate different named xml files which then might accidentally be checked into the repo.